Gemfile

```diff
@@ -1,8 +1,9 @@
 source 'https://rubygems.org'
 
-group :test do
```

Comment:
It's good to separate gemfiles into environments and to have a 'test' group for those relating to testing